packages/grid/x-data-grid-generator/src/renderer/renderEditIncoterm.tsx

```diff
@@ -7,7 +7,7 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import { INCOTERM_OPTIONS } from '../services/static-data';
 
-function EditIncoterm(props: GridRenderEditCellParams<string | null>) {
+function EditIncoterm(props: GridRenderEditCellParams<any, string | null>) {
```

Comment:
What is the motivation behind inverting generics' order?
I feel like in `GridRenderEditCellParams` specifically the `V` generic is the most useful, and because `R` is now first - we have to pass `any` everywhere. The DX feels a bit awkward

Reply:
It's mainly for consistency. Others were typed in a similar manner.